tasks/task444_com_qa_question_paraphrases_answer_generation.json

```diff
@@ -6,7 +6,10 @@
         "https://huggingface.co/datasets/com_qa"
     ],
     "Categories": [
-        "Answer Generation"
+        "Question Answering -> Contextual Question Answering -> Abstractive"
```

Comment:
I think it's not Contextual Question Answering. It seems to be "Question Answering -> Open Question Answering".